dev-docs/creating_bindings.md

```diff
@@ -13,3 +13,11 @@ Keeping similar names and purposes for functions can make it easier for your use
 ## Community
 
 When ready, make a [pull request](https://github.com/uber/h3/edit/master/docs/community/bindings.md) to add your binding to the H3 documentation website.
+
+## Documentation
+
+To be included in the H3 [API reference](https://uber.github.io/h3/#/documentation/api-reference/), your binding should:
+
+* Be reasonably up to date with uber/h3.
+* Include bindings for the functions in listed in `binding-functions`.
```

Comment:
...in the output of `make binding-functions`?

Reply:
Ok